action_files/test_fit_predict.py

```diff
@@ -4,22 +4,74 @@
 from statsforecast import StatsForecast
 from statsforecast.utils import AirPassengersDF as df
 from statsforecast.models import (
-    AutoARIMA, AutoETS, AutoCES, AutoTheta,
-    MSTL, GARCH, ARCH, HistoricAverage,
-    Naive, RandomWalkWithDrift,
-    SeasonalNaive, WindowAverage, SeasonalWindowAverage,
-    ADIDA, CrostonClassic,
-    CrostonOptimized, CrostonSBA, IMAPA,
-    TSB
+    AutoARIMA,
+    AutoETS,
+    AutoCES,
+    AutoTheta,
+    MSTL,
+    GARCH,
+    ARCH,
+    HistoricAverage,
+    Naive,
+    RandomWalkWithDrift,
+    SeasonalNaive,
+    WindowAverage,
+    SeasonalWindowAverage,
+    ADIDA,
+    CrostonClassic,
+    CrostonOptimized,
+    CrostonSBA,
+    IMAPA,
+    TSB,
 )
 
-def get_data():
+
+def get_data(rename_cols: bool = False):
     df2 = df.copy(deep=True)
-    df2['unique_id'] = 2.0
-    df2['y'] *= 2
-    return pd.concat([df, df2])
+    df2["unique_id"] = 2.0
+    df2["y"] *= 2
+    panel_df = pd.concat([df, df2])[["unique_id", "ds", "y"]]
+    if rename_cols:
+        renamer = {
+            "unique_id": "item_id",
+            "ds": "timestamp",
+            "y": "target",
+        }
+        panel_df.rename(columns=renamer, inplace=True)
+    return panel_df
+
 
-@pytest.mark.parametrize('n_jobs', [-1, 1])
+@pytest.mark.parametrize("n_jobs", [-1, 1])
+def test_custom_cols(n_jobs):
+    AirPassengersPanel = get_data(rename_cols=True)
```

Comment:
why `PascalCase` if it is an instance of a pandas data frame? are `PascalCase` not used just for class names?